CHANGELOG/CHANGELOG-1.20.md

```diff
@@ -70,10 +70,6 @@
     - [Added](#added-2)
     - [Changed](#changed-2)
     - [Removed](#removed-2)
-  - [Dependencies](#dependencies-3)
```

Comment:
I think removing the dependencies section below is going to break the dependency links in the subsequent sections, so they likely need to be updated (i.e. #dependencies-4 would now be #dependencies-3)

Reply:
Thank you very much @hasheddan, fixed.